EditorDelegates/EditorDelegatesUser.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace Unity.Labs.SuperScience
+{
+    public class EditorDelegatesUser : MonoBehaviour
```

Comment:
Some kind of description of this class please.  Super Science is meant to be a learning resource so you can err on the side of commenting too much